services/galley/test/integration/API/Teams.hs

```diff
@@ -1008,13 +1007,172 @@ testUpdateTeam = do
     checkTeamUpdateEvent tid u wsOwner
     checkTeamUpdateEvent tid u wsMember
     WS.assertNoEvent timeout [wsOwner, wsMember]
-  where
-    checkTeamUpdateEvent tid upd w = WS.assertMatch_ timeout w $ \notif -> do
-      ntfTransient notif @?= False
-      let e = List1.head (WS.unpackPayload notif)
-      e ^. eventType @?= TeamUpdate
-      e ^. eventTeam @?= tid
-      e ^. eventData @?= Just (EdTeamUpdate upd)
+
+testTeamAddRemoveMemberAboveThresholdNoEvents :: HasCallStack => TestM ()
```

Comment:
The reason for this was a practical. I could easily do that but creating such a team does take some time. This test already takes 30 secs, let me see if it can be improved.

Reply:
Wow, 30 seconds is a bit too much. Maybe we should figure out unit testing soon. As most of this could be tested at unit level.